webapp/src/main/webapp/news-list-view/js/NewsListService.js

```diff
@@ -14,24 +14,12 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-import {newsConstants} from '../../js/newsConstants.js';
 
 export function getNewsList(newsTarget, limit) { // eslint-disable-line no-unused-vars
   // TODO
   return Promise.resolve([]);
 }
 
-export function getNewsTarget() {
```

Comment:
since this function is nomore a part of NewsListService, the call should be modified from news list portlet ?

Reply:
already changed here https://github.com/exoplatform/news/pull/352/files#diff-9b7b97f7f33812a8b3cae3b547ba154c27823e770fa0d7edd6ae307897625b98L201